main.gms

```diff
@@ -658,7 +658,7 @@ parameter
 parameter
   cm_prtpScen               "pure rate of time preference standard values"
 ;
-  cm_prtpScen         = 3;         !! def = 3  !! regexp = 1|3
+  cm_prtpScen         = 1;         !! def = 3  !! regexp = 1|3
```

Comment:
please adjust the default value

Reply:
Default value adjusted. CodeCheck done: [ FAIL 0 | WARN 0 | SKIP 6 | PASS 83 ]
Switch cm_prtpScen obviously not used in scenario_config files.